From f57384fd0a8669c50014382f7f90ed7cba9ba3c8 Mon Sep 17 00:00:00 2001
From: Grigori Goronzy <greg@blackbox>
Date: Tue, 8 Jan 2013 00:33:47 +0100
Subject: [PATCH] Reduce aggressiveness of KSM scanning

The default is too much for older devices, and leads to notable
additional CPU load and power drain.

Change-Id: Ib7658d23152e6674a8b56cdcd6be14f75737c07c
---
 prebuilt/common/etc/init.local.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/prebuilt/common/etc/init.local.rc b/prebuilt/common/etc/init.local.rc
index 521f931..1b62be3 100644
--- a/prebuilt/common/etc/init.local.rc
+++ b/prebuilt/common/etc/init.local.rc
@@ -25,8 +25,8 @@ on boot
     chmod 0664 /sys/kernel/mm/ksm/sleep_millisecs
     chown root system /sys/kernel/mm/ksm/run
     chmod 0664 /sys/kernel/mm/ksm/run
-    write /sys/kernel/mm/ksm/sleep_millisecs 1500
-    write /sys/kernel/mm/ksm/pages_to_scan 256
+    write /sys/kernel/mm/ksm/sleep_millisecs 2000
+    write /sys/kernel/mm/ksm/pages_to_scan 128
 
 # adb over network
 on property:service.adb.tcp.port=5555
-- 
1.8.0

